native/acter/src/api/calendar_events.rs

```diff
@@ -237,6 +229,26 @@ impl CalendarEvent {
             })
             .await?
     }
+
+    pub async fn my_rsvp_status_str(&self) -> Result<String> {
+        let client = self.client.clone();
+        let event_id = self.inner.event_id().to_owned();
+        let my_id = self.client.user_id().context("User not found")?;
+
+        RUNTIME
+            .spawn(async move {
+                let manager =
+                    models::RsvpManager::from_store_and_event_id(client.store(), &event_id).await;
+                // read last rsvp
```

Comment:
just saw below that `rsvp_manager.my_status` exists. Why not use that?
```
                        return rsvp_manager.my_status().await?.text();
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/979/commits/f6a9a779b99fbfacd38031c7fbebd2825e9eb2ca